spec/ParseGeoPoint.spec.js

```diff
@@ -24,7 +24,7 @@ describe('Parse.GeoPoint testing', () => {
         });
       }
     });
-  });
+  }, 60000);
```

Comment:
Pretty sure we have a global 2s timeout for each test already, defined at the top of `helper.js`